### ISSUE TITLE
Plotting Loss Values for each Epoch

### ISSUE THREAD

**Issue by sourav1312** [Accenture/AmpliGraph#230]
Hello!

I am training a knowledge graph model using ampligraph 1.3.2. Could you please let me know how I can save the average loss values for each epoch and plot them in a graph?

Thanks in advance.

**Comment by lukostaz** [comment]
Hey @sourav1312,
Although current codebase does not have that functionality baked in, [you can have a go with TensorBoard for tf 1.x.](https://github.com/tensorflow/tensorboard/blob/master/docs/r1/summaries.md)

**Comment by sourav1312** [comment]
@lukostaz For tracking loss and other metrics using Tensorboard or WandB, the appropriate variables need to be passed as parameters. I could not find any such variable in model.fit() function. How can I refer to such variables for the model that is being trained, like val_loss, train_loss, etc?

**Comment by sumitpai** [comment]
@sourav1312 Currently this is not supported, but we are working on this, and it will be added soon
